Index: src/main/java/polimi/ingsoft/Place.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package polimi.ingsoft;\r\n\r\npublic class Place {\r\n    Deck deck;\r\n    Card places[]=new Card[2];\r\n    public Place(){\r\n        deck=new Deck();\r\n\r\n    }\r\n    public Card getCard(int pos) throws IndexOutOfBoundsException{\r\n        Card temp= deck.draw();\r\n        if(pos==0)return temp;\r\n        Card temp2=places[pos];\r\n        places[pos]=temp;\r\n        return temp2;\r\n        }\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/polimi/ingsoft/Place.java b/src/main/java/polimi/ingsoft/Place.java
--- a/src/main/java/polimi/ingsoft/Place.java	
+++ b/src/main/java/polimi/ingsoft/Place.java	
@@ -5,7 +5,6 @@
     Card places[]=new Card[2];
     public Place(){
         deck=new Deck();
-
     }
     public Card getCard(int pos) throws IndexOutOfBoundsException{
         Card temp= deck.draw();
@@ -14,5 +13,4 @@
         places[pos]=temp;
         return temp2;
         }
-    }
 }
